Branch-TestBed/Branch-TestBed.xcodeproj/project.pbxproj

```diff
@@ -208,7 +203,7 @@
 				ORGANIZATIONNAME = "Branch Metrics";
 				TargetAttributes = {
 					6700165F1940F51400A9E103 = {
-						DevelopmentTeam = 6B2DFWM843;
```

Comment:
You can't prevent Xcode from changing it (unless you have that team available in Xcode), but you can certainly just not commit ;)